plugins/meson.build

```diff
@@ -1,6 +1,7 @@
 subdir('acpi-dmar')
 subdir('acpi-facp')
 subdir('ccgx')
+subdir('cros_ec-usb')
```

Comment:
same here. Git rewrite to the rescue?

Reply:
So just to be clear, cros_ec-usb -> cros-ec?

Reply:
yup

Reply:
Done.